p2p/utils_test.go

```diff
@@ -81,111 +66,8 @@ func validatorId(i int) string {
 	return fmt.Sprintf(serviceURLFormat, i)
 }
 
-func waitForNetworkSimulationCompletion(t *testing.T, wg *sync.WaitGroup) {
```

Comment:
All of these helpers are still being used, their refactoring hasn't been merged yet. I'm wondering if the linter isn't seeing these imports because the only consumer (p2p/module_raintree_test.go) includes the `//go:build test` tag. This got me to notice that I neglected to add the `tag` to the CI command which I've addressed in #802. (Still waiting on CI to see if this has any impact on the linter yet.)

Reply:
That doesn't seem to have any effect on the linter. I took a quick look at `golangci-lint --help` and it mentions a `--build-tags` flag:
```
Run the linters

Usage:
  golangci-lint run [flags]

Flags:
      ...
      --build-tags strings             Build tags
```

Reply:
I updated #802 to add `--build-tags` to the linter call but what confuses me is that the linter seemed to succeed in the commit prior (where only the `go test` call was changed). :thinking:

Reply:
@bryanchriswhite Can you update the Makefile / tooling appropriately to handle this?

Reply:
@Olshansk jinx?

Reply:
I also don't seem to be able to reproduce the same linter output as CI, locally. Doesn't matter whether I use `make go_lint` or `golangci-lint run`.

Reply:
I was able to repro locally with `make go_lint`.

@adshmh Now that @bryanchriswhite's change is merged in, see if you're still blocked